### ISSUE TITLE
More of a request related to custom designs.

### ISSUE THREAD

**Issue by jdroo** [MyShiLingStar/ACNHPoker#27]
**Can you maybe possibly add a way to edit custom designs and pro designs? specifically a way to unlock them so I can edit them in game?**

I've been unable to find any existing **updated** tool that can do this one task...

NLSE can import and export them, tho even it seems to be unable to transfer ownership of the designs to me...

Maybe just add a way to view each existing design, and have a button to click on the ones you want to "unlock" so you can edit them in game, 

I mostly only want to edit the ownership of designs I've made on https://acpatterns.com/editor that were sent to my game via the Nintendo owned phone app. 

_[kinda not related, but I wish someone made a bot of sorts, that uses sys_botbase, that can read the files exported from acpatterns.com, and have it automate "drawing" the designs in game, that would be heckin' neat! lol]_

**Comment by TheSmilez** [comment]
There’s already something for this.

https://github.com/FluffyFishGames/ACNHDesignPatternEditor

Uses the save file, instead.

I do agree it would be cool if it could be done on the fly, but I don’t think ACNH Poker is or should be for that.

**Comment by jdroo** [comment]
Yeah he updated that recently, but even it can't unlock the custom designs to be edited in game. oof. oh well, thanx for the input!

**Comment by moddedx** [comment]
> Yeah he updated that recently, but even it can't unlock the custom designs to be edited in game. oof. oh well, thanx for the input!

Designs are lock for a reason that are not your. If you want to edit locked designs maybe learn some coding and other things. Stop asking for things that could steal other people's hard work.

**Comment by TheSmilez** [comment]
> Yeah he updated that recently, but even it can't unlock the custom designs to be edited in game. oof. oh well, thanx for the input!

You can edit your current designs in the Pattern Editor from GitHub and then just change the name. When you start up your game you will then be able to edit them. Done this multiple times to create different angles from some of my favorite paths.

**Comment by jdroo** [comment]
> > Yeah he updated that recently, but even it can't unlock the custom designs to be edited in game. oof. oh well, thanx for the input!
> 
> You can edit your current designs in the Pattern Editor from GitHub and then just change the name. When you start up your game you will then be able to edit them.

Yeah for some reason that's not working for me, But when I import a new design, it sets the name correct, but remains locked. not sure why that's happening.

**Comment by MyShiLingStar** [comment]
Poker had another fork that has the custom designs thingy before. Not sure if that guy is still working on it or not.
I heard someone was using it to "take other people's designs and claim as their own". And got into trouble with the developer.


In other words, the feature would raise ethical questions similar to dream island dumping.
I most likely would not work on it unless I find an interest in it.
